entry_types/scrolled/package/src/entryState/sharing.js

```diff
@@ -31,7 +31,7 @@ import WhatsAppIcon from "../frontend/assets/images/navigation/icons/social/what
  *     }
  *   ]
  */
-export function useShareProviders() {
+export function useShareProviders(isPhone) {
```

Comment:
One last thing that I missed earlier. Let's use an options object here, so that the call looks like `useShareProvider({isPhonePlatform: true})` instead of `useShareProvider(true)`. In the latter case you have to look at the definition of the hook to find out what the `true` means.